kiwi/kiwi_compat.py

```diff
@@ -189,22 +187,21 @@ def __repo_argument(self, source, repo_type, alias, priority):
 class Command(object):
     @classmethod
     def execute(self, arguments):
-        # legacy kiwi binary lives in /usr/sbin, the new kiwi
-        # lives in /usr/bin. In order to avoid conflicts the path
-        # to the kiwi we want to use is specificly set here
-        kiwi = '/usr/bin/kiwi'
+        kiwi = '/usr/bin/kiwi-ng'
```

Comment:
Just some nitpicking here: if someone calls `./setup.py --home` or `./setup.py --prefix SOME_PATH` this would probably break.
Not sure how to solve it. *scratching my head*